iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/node/ClusterNodeStartUtils.java

```diff
@@ -314,6 +315,25 @@ public static TSStatus confirmNodeRestart(
         break;
     }
 
+    // check clusterId if not empty
+    if (clusterId != null
+        && !clusterId.isEmpty()
+        && !clusterId.equals(configManager.getClusterManager().getClusterId())) {
+      status.setCode(TSStatusCode.REJECT_NODE_START.getStatusCode());
+      status.setMessage(
+          String.format(
+              "Reject %s restart. Because the clusterId of the current %s and the target cluster are inconsistent. "
+                  + "ClusterId of the current Node: %s, ClusterId of the target cluster: %s."
+                  + POSSIBLE_SOLUTIONS
+                  + "\t1. Change the cluster_id parameter in %s to match the target cluster.",
```

Comment:
“Solution" might be not proper here, because we don't want user to modify system.properties. Maybe just suggest the user to check if the node configuration is correct?

Reply:
Fixed